src/System.Net.Primitives/tests/UnitTests/CookieInternalTest.cs

```diff
@@ -68,5 +68,94 @@ public void CookieParserGetServer_SetCookieHeaderValue_Success(string cookieStri
             int expectedCookieCount = expectedStrings.Length >> 1;
             Assert.Equal(expectedCookieCount, cookieCount);
         }
+
+        // This assumes that the default cookie behavior is RFC 6265, which it
+        // is not ("Default = Rfc2109" in Cookie.cs:20)
+        // TODO: Will the Default behavior change to RFC 6265? If not, specify appropriate CookieVariant
+        [Theory]
+        [InlineData("https://contoso.com/", "/")]
+        [InlineData("https://contoso.com", "/")]
+        [InlineData("https://contoso.com/path", "/")]
+        [InlineData("https://contoso.com/path/subpath", "/path")]
+        [InlineData("https://contoso.com/path/subpath/", "/path/subpath")]
+        [InlineData("https://contoso.com/path/subpath?query=queryString", "/path")]
+        public void SetCookies_Sets_Rfc6265_Default_Path_From_Url(string url, string expectedPath)
+        {
+            var uri = new Uri(url);
+            var cc = new CookieContainer();
+            cc.SetCookies(uri, "name=value");
+
+            Assert.Equal(expectedPath, cc.GetCookies(uri)["name"].Path);
```

Comment:
Instead of using `"name"`, we can [create some `const string`.](https://github.com/dotnet/corefx/pull/33287/files#diff-bef05ae5b84d8a8bfe84c367e3e4d76fR14)